manifests/pcidevices-operator.yaml

```diff
@@ -0,0 +1,124 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: pcidevices
+    app.kubernetes.io/version: master-head
+  name: pcidevices
+rules:
+  - apiGroups: [ "apiextensions.k8s.io" ]
+    resources: [ "customresourcedefinitions" ]
+    verbs: [ "*" ]
+  - apiGroups: [ "" ]
+    resources: [ "nodes" ]
+    verbs: [ "get", "watch", "list", "update" ]
+  - apiGroups: [ "" ]
+    resources: [ "configmaps", "events", "secrets"]
+    verbs: [ "get", "watch", "list", "update", "create" ]
+  - apiGroups: [ "" ]
+    resources: [ "namespaces" ]
+    verbs: [ "get", "watch", "list" ]
+  - apiGroups: [ "devices.harvesterhci.io" ]
+    resources: [ "pcidevices", "pcidevices/status", "pcideviceclaims", "pcideviceclaims/status" ]
+    verbs: [ "get", "watch", "list", "update", "create", "delete"]
+  - apiGroups: ["admissionregistration.k8s.io"]
+    resources: ["mutatingwebhookconfigurations"]
+    verbs: [ "get", "watch", "list", "update", "create", "delete" ]
+  - apiGroups: ["apiregistration.k8s.io"]
+    resources: ["apiservices"]
+    verbs: [ "get", "watch", "list", "update", "create", "delete" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    app.kubernetes.io/name: pcidevices
+    app.kubernetes.io/version: master-head
+  name: pcidevices
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pcidevices
+subjects:
+  - kind: ServiceAccount
+    name: pcidevices
+    namespace: harvester-system
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    app.kubernetes.io/name: pcidevices
+    app.kubernetes.io/version: master-head
+  name: pcidevices
+  namespace: harvester-system
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: pcidevices
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: pcidevices
+        app.kubernetes.io/version: master-head
+    spec:
+      containers:
+        - args:
+            - agent
+          env:
+            - name: NODE_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+          image: rancher/harvester-pcidevices:master-head
+          imagePullPolicy: Always
+          name: agent
+          resources:
+            limits:
+              memory: 200Mi
+            requests:
+              cpu: 10m
+              memory: 50Mi
+          securityContext:
+            privileged: true
+          volumeMounts:
+            - mountPath: /lib/modules
+              name: modules
+            - mountPath: /sys
```

Comment:
I'm curious how this worked before without the /sys mount

Reply:
It might sound crazy, but it was working thanks to `hostNetwork: true`!